maps/delta/outfits/guardsmen.dm

```diff
@@ -211,4 +230,61 @@ Begin Warhammer loadouts
 
 	pda_slot = null
 
-	flags = OUTFIT_NO_BACKPACK|OUTFIT_NO_SURVIVAL_GEAR
\ No newline at end of file
+	flags = OUTFIT_NO_BACKPACK|OUTFIT_NO_SURVIVAL_GEAR
+
+/// Commissar 
+
+/decl/hierarchy/outfit/job/ig/commissar
+	name = OUTFIT_JOB_NAME("Commissar")
+	uniform = /obj/item/clothing/under/color/black
+	suit = /obj/item/clothing/suit/armor/commissar
+	back = /obj/item/storage/backpack/satchel/warfare
+	belt = /obj/item/material/sword/commissword
+	gloves = /obj/item/clothing/gloves/combat/cadian
+	shoes = /obj/item/clothing/shoes/jackboots/cadian
+	head = /obj/item/clothing/head/commissar
+	mask = null
+	glasses = null
+	l_pocket = /obj/item/storage/box/ifak
+	r_pocket = /obj/item/device/binoculars
+	suit_store = /obj/item/gun/projectile/bolter_pistol
+	neck = /obj/item/reagent_containers/food/drinks/canteen
+	l_ear = /obj/item/device/radio/headset/red_team
+	backpack_contents = list(
+	/obj/item/ammo_magazine/bolt_pistol_magazine = 2,
+	/obj/item/grenade/smokebomb = 1,
+	/obj/item/device/flashlight/lantern = 1,
+	/obj/item/reagent_containers/food/snacks/warfare/rat = 1,
+	/obj/item/stack/thrones/five = 1,
+	/obj/item/stack/thrones2/five = 1,
+	/obj/item/stack/thrones3/five = 1,
+	)
+
+	pda_slot = null
+
+	flags = OUTFIT_HAS_BACKPACK|OUTFIT_NO_SURVIVAL_GEAR
+
+//// TEMPLATE ///
```

Comment:
The code comments ("//") seem to be messing up the tabbing here, you can comment out a section alternatively like this:
`/*
Code Comment
*/`

Reply:
Is it? Then i will resolve it.